Formula/emscripten.rb

```diff
@@ -25,7 +25,7 @@ class Emscripten < Formula
 
   depends_on "cmake" => :build
   depends_on "binaryen"
-  # error "fatal error: '__config' file not found" when building llvm 12 on High Sierra
+  depends_on "libffi"
```

Comment:
I think this can be `uses_from_macos`

Reply:
I thought so too, but the llvm formula has `depends_on`, not `uses_from_macos`. Thoughts?

Reply:
Wait, I just realised I also took out `-DLLVM_ENABLE_FFI=ON`, so this shouldn't even be linking with libffi at all. I'll just remove all references to ffi.